Makefile

```diff
@@ -333,7 +348,8 @@ test_c_api_rs: install_rs_check_toolchain
 		c_api
 
 .PHONY: test_c_api_c # Run the C tests for the C API
-test_c_api_c: build_c_api
+test_c_api_c:
+	@FORWARD_COMPAT=forward_compatibility "$(MAKE)" build_c_api
```

Comment:
Is there no case in which we want to run tests on C API without the forward compatibility?

Reply:
no you are right it would be better, I'll update

Reply:
this little maneuver cost us 51 years 😂 but it was necessary